README.org

```diff
@@ -42,34 +42,36 @@ You may install some of the stylesheets directly in your browser from [[http://u
 
 Install the stylesheet of your choice according to your browser's method (e.g. using the Stylish extension in Firefox).
 
+Multiple themes besides solarized are supported. The currently supported themes are listed in =css/=. After selecting a theme, the css files you need will be inside.
+
 **** Files
 
-- ~solarized-all-sites-*.css~
+- ~theme-all-sites-*.css~
   These have all the sites' styles smushed into one big file.  Might work pretty well for most sites.  If the collection grows, it might begin to have conflicts--but it still might work pretty well.  It might be a lot better than adding a separate stylesheet for every site you visit...
-- ~solarized-generic-*.css~
+- ~theme-generic-*.css~
   These are intended to be generic, suitable for many simple sites.  They probably won't do much for fancy, popular sites with lots of custom CSS, but for simple, mostly unstyled sites, they might work okay.
-- ~solarized-mediawiki-*.css~
+- ~theme-mediawiki-*.css~
   For MediaWiki sites, like Wikipedia
-- ~solarized-disqus-*.css~
+- ~theme-disqus-*.css~
   For Disqus comments
-- ~solarized-github-*.css~
+- ~theme-github-*.css~
   For GitHub
-- ~solarized-google-*.css~
+- ~theme-google-*.css~
   For Google
-- ~solarized-hackernews-*.css~
+- ~theme-hackernews-*.css~
   For Hacker News
-- ~solarized-planet.emacsen.org-*.css~
+- ~theme-planet.emacsen.org-*.css~
   For [[http://planet.emacsen.org][Planet Emacsen]]
-- ~solarized-stackexchange-*.css~
+- ~theme-stackexchange-*.css~
   For StackExchange sites.  Not all sites are covered, but some of the main ones are.  Patches welcome.
-- ~solarized-sakai-*.css~
+- ~theme-sakai-*.css~
   For sites hosted by [[https://sakaiproject.org/][sakai]], a popular course management program.
 
 ** Development                                                  :noexport_1:
 
 To make changes, just edit the =.styl= files and run =make=.  (You need to have Stylus installed, of course.)
 
-Basically, nearly the only things that should be in =dark.styl= or =light.styl= should be changes to the =color-= variables that adjust it for light or dark.  Everything else should go in =sites/*.styl=.  Colors are defined as =color-= variables, and mixins are used to insert common CSS properties (like =color=, =background=, etc) with =!important=.  Most changes can be made without inserting CSS properties directly into the selectors.
+Basically, nearly the only things that should be in =themes/*/*.styl= should be changes to the =color-= variables that adjust it for light or dark.  Everything else should go in =sites/*.styl=.  Colors are defined as =color-= variables, and mixins are used to insert common CSS properties (like =color=, =background=, etc) with =!important=.  Most changes can be made without inserting CSS properties directly into the selectors.
```

Comment:
I think we need to replace the "light/dark" language since this isn't specific to Solarized anymore.  How about something like:

```
Basically, nearly the only things that should be in =themes/*/*.styl= should be theme-specific colors.
```